src/App.js

```diff
@@ -1,7 +1,18 @@
 import React from 'react'
+import ContainerExampleAlignment from './Components/Layout/index.js'
+
+/**
+ *
+ *Dropdown Menu
+ * Chart
+ */
 
 function App () {
-  return <div className='App'>Hello World!</div>
+  return (
+    <div className='App'>
+      <ContainerExampleAlignment />
```

Comment:
Perhaps we should rename this component to exclude "Example" fromthe name? That almost sounds like a test feature rather than one we plan to build off of.

Reply:
Not a big deal though.